### DISCUSSION TITLE
zh/Q-and-A/answer/question4

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#917] [Announcements]
# zh/Q-and-A/answer/question4

为什么#include iostream 找不到 问题描述： 我遇到了，#include iostream 找不到，路径调配了gcc，并且#include stdio.h 是可以正常使用的，我问了ai，它是这个是gcc里标准库，让我查路径，路径也没问题，无法解决，请问如何处理

https://cygnomatic-controller.github.io/zh/Q-and-A/answer/question4.html

**Comment by GzqHerry** [comment]
你这是.c文件，iostream是C++的特有库。你将文件名后缀改为.cpp
